File: Assets/Scripts/DTO/Request/Exercise/CompletedFireflyExerciseDTO.cs
using DTO.Request.Exercise.@base;

namespace DTO.Request.Exercise
{
    public class CompletedFireflyExerciseDTO : CompletedExerciseDTO
    {
        public int caughtFirefliesCount;
        public int caughtWrongFirefliesCount;
        // TODO: Add more data to store
    }
}

File: backend/src/main/java/vr/balance/app/DTO/request/CompletedFireflyExerciseDTO.java
    package vr.balance.app.DTO.request;

    import lombok.*;
    import lombok.experimental.SuperBuilder;
    import vr.balance.app.DTO.request.base.CompletedExerciseDTO;


    @EqualsAndHashCode(callSuper = true)
    @Data
    @SuperBuilder
    @NoArgsConstructor
    @AllArgsConstructor
    public class CompletedFireflyExerciseDTO extends CompletedExerciseDTO {
        int caughtFirefliesCount;
        int caughtWrongFirefliesCount;
    }
